src/main/common/components/ccj/paid-amount-summary.njk

```diff
@@ -10,9 +10,11 @@
 
     <div class="form-group">
       {% set total = claim.totalAmountTillToday if amountSettledFor === undefined else amountSettledFor + (claimFee | pennies2pounds) %}
-
-      <p>{{ t('The judgment will order the defendant to pay {{ totalAmount }}, including your claim fee and any interest.', { totalAmount: total | numeral }) }}</p>
-      <p>{{ t('This means you´ll recover the {{ totalAmount }} as follows:', { totalAmount: total | numeral }) }}</p>
+      {% if interestDetails %}
+        <p>{{ t('The judgment will order the defendant to pay {{ totalAmount }}, including your claim fee and any interest, as shown in this table:', { totalAmount: total | numeral }) }}</p>
+      {% else %}
+        <p>{{ t('The judgment will order the defendant to pay {{ totalAmount }}, including your claim fee, as shown in this table:', { totalAmount: total | numeral }) }}</p>
```

Comment:
are these 2 of statements same? or am i missing something?

Reply:
yes, its different "and any interest" is additional text for claim with intrest